fix-whitespace.yaml

```diff
@@ -7,5 +7,4 @@ included-files:
   - "*.md"
 
 excluded-files:
-  - "README/Text/Tabular.agda"
-  - CHANGELOG.md
```

Comment:
AFAICT, this exclusion was added in commit by e0bd51a8c487075994a681309b62078328976adb
by @MatthewDaggitt but not sure why!?

Reply:
Weirdly the PR that it is attached to points to commit [168f7e3](https://github.com/agda/agda-stdlib/pull/1379/commits/168f7e3d1cdbb0f7c97072f5bc58f87679dc2831) instead. It looks like this just 'snuck in' without discussion.